src/app/lib/config/services/arabic.js

```diff
@@ -1,110 +1,114 @@
 import { C_POSTBOX } from '@bbc/psammead-styles/colours';
-import { latin } from '@bbc/gel-foundations/scripts';
+import { arabic } from '@bbc/gel-foundations/scripts';
 import { news as brandSVG } from '@bbc/psammead-assets/svgs';
 
 const service = {
-  lang: `ar-SA`,
-  articleAuthor: `http://www.facebook.com/bbcarabic`,
-  articleTimestampPrefix: 'Updated',
+  lang: 'ar',
+  product: 'BBC News',
+  articleAuthor: 'https://www.facebook.com/bbcnews',
+  articleTimestampPrefix: 'جدّد في',
   atiAnalyticsAppName: 'news-arabic',
-  brandName: 'BBC News Arabic',
-  product: 'BBC News Arabic',
+  brandName: 'BBC News عربي',
+  serviceLocalizedName: 'عربي',
   defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/arabic.png',
-  defaultImageAltText: 'BBC News Arabic',
-  dir: `ltr`,
-  externalLinkText: ', external',
-  imageCaptionOffscreenText: 'Image caption, ',
-  videoCaptionOffscreenText: 'Video caption, ',
-  audioCaptionOffscreenText: 'Audio caption',
-  defaultCaptionOffscreenText: 'Caption, ',
-  imageCopyrightOffscreenText: 'Image source, ',
-  locale: `ar-SA`,
-  datetimeLocale: `ar-SA`.toLowerCase(),
+  defaultImageAltText: 'BBC News عربي',
+  dir: 'rtl',
+  externalLinkText: 'خارجي ،',
+  imageCaptionOffscreenText: '،التعليق على الصورة',
+  videoCaptionOffscreenText: '،التعليق على الفيديو',
+  audioCaptionOffscreenText: '،التعليق على التسجيل السماعي',
+  defaultCaptionOffscreenText: '،التعليق',
+  imageCopyrightOffscreenText: '،مصدر الصورة',
+  locale: 'ar',
+  datetimeLocale: 'ar',
   service: 'arabic',
-  serviceName: 'News Arabic',
+  serviceName: 'Arabic',
   themeColor: `${C_POSTBOX}`,
   twitterCreator: '@BBCArabic',
   twitterSite: '@BBCArabic',
   noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
   publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
-  script: latin,
+  script: arabic,
   manifestPath: '/articles/manifest.json',
-  swPath: '/articles/sw.js',
+  swPath: '/sw.js',
   translations: {
     error: {
       404: {
         statusCode: '404',
```

Comment:
```suggestion
        statusCode: '۴۰۴',
```

Reply:
Resolving for this reason: https://github.com/bbc/simorgh/pull/2706#discussion_r307236205